public/app/features/alerting/unified/components/rule-viewer/RuleViewer.test.tsx

```diff
@@ -1,16 +1,26 @@
+import 'whatwg-fetch';
```

Comment:
This shouldn't be needed now that https://github.com/grafana/grafana/pull/85578 was merged